libyul/AsmAnalysis.cpp

```diff
@@ -272,14 +286,14 @@ vector<YulString> AsmAnalyzer::operator()(FunctionCall const& _funCall)
 	auto watcher = m_errorReporter.errorWatcher();
 	vector<YulString> const* parameterTypes = nullptr;
 	vector<YulString> const* returnTypes = nullptr;
-	vector<bool> const* needsLiteralArguments = nullptr;
+	vector<std::optional<LiteralKind>> const* literalArguments = nullptr;
```

Comment:
```suggestion
	vector<optional<LiteralKind>> const* literalArguments = nullptr;
```
Yeah, I know, it's annoying :-).